viz/src/machines/viz.ts

```diff
@@ -283,7 +396,9 @@ export const vizMachine =
           );
           const rawResponseBody = await response.json();
```

Comment:
The server might be sending back http code status 500 failure that does not contain any body. Unless i'm mistaken, it will result to display "unknown server error" inside the client right ?